src/translate/translate.js

```diff
@@ -1579,7 +1579,9 @@ export const processFilterArgument = ({
   const filterParamKey =
     paramIndex > 1 ? `_${paramIndex - 1}_${argumentName}` : argumentName;
   const filterCypherParam = `$${filterParamKey}`;
-  let translations = [];
+  let translations = process.env.SKIP_INJECT
```

Comment:
can you add a link to the JIRA ticket as a comment here?
```
// [ITG-XXX] need to filter out placeholders
```

Reply:
Done